include/Chibi.hpp

```diff
@@ -45,8 +46,8 @@ class Chibi {
          * using the #make_from function. The result will be converted from C++ back to scheme using the #SExp.to
          * function.
          */
-        template<typename Return, typename... Args>
-        ClassRegistrator<Class> &register_method(std::string &&name, Return (Class::*member_function)(Args...));
+        template <typename Return, typename... Args>
+        ClassRegistrator<Class> &register_method(const std::string &&name, Return (Class::*member_function)(Args...));
```

Comment:
const &&-reference?
I don't think this is right, we will just deep copy contents from const reference, since move will be impossible.